src/redux/categories/categories.jsx

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] Kindly make sure when the check status button is clicked in your [categories.jsx](https://github.com/ambrose-kibet/react-bookstore/blob/22ac96af4378a022e0a8fc770a7b4f5775a78435/src/pages/Categories.jsx#L1) , it dispatches the underConstruction from your action creator in categoriesSlice